app/script/view_model/content/MessageListViewModel.js

```diff
@@ -636,7 +636,7 @@ z.viewModel.content.MessageListViewModel = class MessageListViewModel {
     const isUnreadMessage = messageTimestamp > conversationTimestamp;
     const isNotOwnMessage = !messageEntity.user().is_me;
     const expectsConfirmation = messageEntity.expectsReadConfirmation;
-    const shouldSendReadReceipt = this.conversation_repository.shouldSendReadReceipt(conversationEntity);
+    const shouldSendReadReceipt = this.conversation_repository.expectReadReceipt(conversationEntity);
```

Comment:
We should remove this condition here. This is checked at receiving time now, no need to recheck at viewing time.

I think with that, the following scenario is not working:
- RR are ON
- A logs out
- B sends msg
- B turns RR OFF
- A logs in

=> I believe A will not send a RR (because the last state of the convo for RR is OFF)

Reply:
Now I remember why we needed this method to return `true` in case of a group + team conversation.

We might want to create another method then...

Reply:
Ok, let's talk about creating another method.